src/main/resources/style/taskcardstyles/TaskCardBaseStyle.css

```diff
@@ -0,0 +1,59 @@
+#taskCard {
+    -fx-fit-to-width: true;
+}
+
+#titleLabel {
+    -fx-font-size: 20pt;
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-text-fill: #2D2D2D;
+    -fx-font-smoothing-type: lcd;
+}
+
+#typeLabel0, #typeLabel1, #tagLabel0, #tagLabel1, #tagLabel2, #tagLabel3, #tagLabel4 {
+    -fx-background-radius: 10;
+    -fx-text-fill: #ffffff;
+    -fx-font-size: 10pt;
+    -fx-font-family: "Segoe UI Semilight";
```

Comment:
Segoe UI is a Windows font. Might want to consider embedding our own font at some point (please open a new issue for this)